src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -568,6 +632,28 @@ private static void handleKeyValueConfig(TestInputConfiguration.Builder testInpu
         }
     }
 
+    public static Map<Object, Object> getDefaultProperties(String filePath, String moduleName)
+            throws CheckstyleException {
+
+        final Map<Object, Object> defaultProperties;
+        final boolean isModuleValid = !moduleName.contains(".")
+            && !SUPPRESSED_MODULES.contains(moduleName);
+
+        if (isModuleValid) {
+            defaultProperties = new HashMap<>();
+            final String fullyQualifiedClassName = getFullyQualifiedClassName(filePath, moduleName);
+            final ModuleDetails moduleDetails = MODULE_DETAILS_MAP.get(fullyQualifiedClassName);
+            moduleDetails.getProperties().forEach(prop -> {
```

Comment:
```suggestion
            defaultProperties = moduleDetails.getProperties().map(.....)
```
actual function(ality) is called `map`, invasive done with `forEach` as this some kind of DTO context.

Try leveraging lambdas feature/embracing functional programming.

Reply:
@Anmol202005 , pleease  do this